src/crsf.h

```diff
@@ -34,6 +34,8 @@
 #define TYPE_SETTINGS_ENTRY   0x2B
 #define TYPE_SETTINGS_READ    0x2C
 #define TYPE_SETTINGS_WRITE   0x2D
+#define TYPE_RADIO_ID         0x3A
+#define TYPE_SYNC             0x10
```

Comment:
Please move TYPE_SYNC out of this list and give it a different prefix.  It's not a frame type (it's not applied to the third received telemetry type).  From the comment in opentx 2.3 it's some kind of sub-type that indicates a "timing correction frame".

Reply:
renamed to `SUBTYPE_TIMING_UPDATE` and moved to a new list with appropriate comment